src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -160,6 +173,11 @@ export default class RegistrationDefineBusiness extends Mixins(CommonMixin) {
       .every(flag => flag)
   }
 
+  onNatureOfBusinessValidEvent (valid: boolean): void {
+    this.natureOfBusinessValid = valid
+    this.setRegistrationDefineBusinessValid(this.allFlagsValid)
+  }
```

Comment:
You know you've got good architecture when adding a new component is this easy 😃